src/iexamine.cpp

```diff
@@ -630,22 +630,65 @@ void iexamine::cardreader(player *p, map *m, int examx, int examy)
 void iexamine::rubble(player *p, map *m, int examx, int examy)
 {
     bool has_digging_tool = p->has_items_with_quality( "DIG", 2, 1 );
-  // Perhaps check for vehicle covering the rubble and bail out if so (string freeze ATM)?
+    // Perhaps check for vehicle covering the rubble and bail out if so (string freeze ATM)?
     if (!has_digging_tool) {
         add_msg(m_info, _("If only you had a shovel..."));
         return;
     }
+
+    // Ask if there's something possibly more interesting than this rubble here
     std::string xname = m->furnname(examx, examy);
-    if (query_yn(_("Clear up that %s?"), xname.c_str())) {
-        // "Remove"
-        p->moves -= 200;
-        m->furn_set(examx, examy, f_null);
+    if( ( m->veh_at( examx, examy ) != nullptr ||
+          m->tr_at( examx, examy ) != tr_null ||
+          g->critter_at( examx, examy ) != nullptr ) &&
+          !query_yn(_("Clear up that %s?"), m->tername(examx, examy).c_str() ) ) {
+        none(p, m, examx, examy);
+        return;
+    }
+
+    // "Remove"
+    p->moves -= 200;
+    m->furn_set(examx, examy, f_null);
+
+    // "Remind"
+    add_msg(_("You clear up that %s."), xname.c_str());
+}
+
+void iexamine::crate(player *p, map *m, int examx, int examy)
+{
+    // Check for a crowbar in the inventory
+    const auto has_prying = []( const item it ) {
+        const auto fun = it.type->get_use( "CROWBAR" );
```

Comment:
> Though then I'd need to handle the cases where the item returns -1.

??? What item should return -1? Do you mean invoking the iuse function returning -1? Your current code already ignores the return value of the isue function.

At most you might need another overload version of `bool player::invoke_item( item* used, const std::string &method )`, one with the point that is forwarded to the iuse function. You can just add that point parameter to the existing function and make a compatibility function that calls the new version with the point being `player::pos()` (kind of a default value for that parameter).

However if you don't care about charges and assume `iuse::crowbar` will never use charges and will never destroy the tool, you can invoke the item returned by `items_with` and forget the return value.